spec/integration/skip_require_patch_spec.rb

```diff
@@ -17,7 +17,14 @@
 
 # frozen_string_literal: true
 
-module ElasticAPM
-  RSpec.describe Context::Request do
+# Deliberately setting this before requiring spec_helper.rb so it's set
+# before we require the agent
+ENV['ELASTIC_APM_SKIP_REQUIRE_PATCH'] = '1'
```

Comment:
Do you want to cache the original value of the env variable and then set it again after this test? I know it's not really necessary but it would just be so that this test is completely contained.

Reply:
As you mention; the integration tests are run in their own processes and setting it like this doesn't `export` it to the outside `env`. I don't want to add a _fix_ for something that isn't in fact broken.

Reply:
It would be an improvement, not a fix :)
I am just always wary of making changes to something global in a single test.